CHANGES.txt

```diff
@@ -60,6 +60,8 @@ Breaking Changes
 Changes
 =======
 
+- Fixed filename in log rotate pattern in ``log4j2.properties``.
```

Comment:
could we phrase this in a way that it's more clear for users what was fixed?

Reply:
@mfussenegger pushed fixup